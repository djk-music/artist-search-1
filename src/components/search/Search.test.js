import React from 'react';
import renderer from 'react-test-renderer';
import Search  from './Search';

jest.mock('../../services/artistsAndWorks.js');

describe('Search', () => {
  it('matches a snapshot', () => {
    const tree = renderer.create(
      <Search />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
