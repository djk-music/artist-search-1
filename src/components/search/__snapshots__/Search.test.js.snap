// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search matches a snapshot 1`] = `
Array [
  <div
    className="header"
  >
    <h1>
      Search
    </h1>
    <form
      onSubmit={[Function]}
    >
      <input
        name="artist"
        onChange={[Function]}
        placeholder="type artist name"
        type="text"
        value=""
      />
      <button
        onClick={[Function]}
      >
        Search
      </button>
    </form>
  </div>,
  <div>
    <div
      className="paging"
    >
      <button
        onClick={[Function]}
      >
        Back
      </button>
      <button
        onClick={[Function]}
      >
        Forward
      </button>
    </div>
    <h3>
      Artists
    </h3>
    <ul />
  </div>,
]
`;
